butterfly-core/src/test/java/com/paypal/butterfly/core/UpgradePathTest.java

```diff
@@ -0,0 +1,39 @@
+package com.paypal.butterfly.core;
+
+import org.testng.annotations.Test;
+import com.paypal.butterfly.extensions.springboot.SpringBootUpgrade_1_5_6_to_1_5_7;
+
+import static org.testng.Assert.*;
+
+/**
+ * UpgradePath Test
+ *
+ * Created by Badal Sarkar on 10/29/2020
+ */
+public class UpgradePathTest {
+
+    @Test
+    public void testUpgradePathIsValid(){
+        SpringBootUpgrade_1_5_6_to_1_5_7 sbu = new SpringBootUpgrade_1_5_6_to_1_5_7();
+        UpgradePath upgradePath = new UpgradePath(SpringBootUpgrade_1_5_6_to_1_5_7.class, "1.5.7");
+        assertTrue(upgradePath instanceof UpgradePath);
+    }
+
+    @Test(expectedExceptions = IllegalArgumentException.class, expectedExceptionsMessageRegExp= "The requested upgraded version is inexistent")
+    public void testUpgradePathNonExsists(){
+        SpringBootUpgrade_1_5_6_to_1_5_7 sbu = new SpringBootUpgrade_1_5_6_to_1_5_7();
+        UpgradePath upgradePath = new UpgradePath(SpringBootUpgrade_1_5_6_to_1_5_7.class, "1.5.9");
```

Comment:
Please remove the variable on the left side of the assignment, since it is not used. Like this:

`new UpgradePath(SpringBootUpgrade_1_5_6_to_1_5_7.class, "1.5.9");``

Reply:
Made necessary changes. Also I removed line 24 as it is not needed. I have to be more careful next time about unused code.